### PR TITLE
docs: updating text and images to match installer rc.

### PR DESCRIPTION
**Pull request by zbwright**

### CODE REVIEW

**Comment by joshix** [comment]
ok to test

**Comment by sym3tri** [comment]
@joshix no need to run tests for docs changes

File: Documentation/tutorials/installing-tectonic.md [hunk 1]
```diff
@@ -138,6 +138,8 @@ Enter the email address and password that will be used to log in to Tectonic Con
 
 Click *Submit* to submit your assets and create your Kubernetes cluster. (Cluster creation may take up to 20 minutes.)
 
+Click *Advanced mode: Manually boot* to validate configuration, generate assets, but not create the cluster.
```
Comment (sym3tri) [inline Documentation/tutorials/installing-tectonic.md]: do we want to link to the "manual boot" doc from here?

Comment (zbwright) [inline Documentation/tutorials/installing-tectonic.md]: ha! we can now.

**Pull request merged**